exampleplugins/5.2/SeoExample/SeoExample.php

```diff
@@ -79,7 +79,9 @@ public function loadPerformanceExtension(\Enlight_Controller_ActionEventArgs $ar
             return;
         }
 
-        $subject->View()->addTemplateDir(__DIR__ . '/Resources/views/');
+        $pluginBasePath = $this->container->getParameter('seo_example.plugin_dir');
+
+        $subject->View()->addTemplateDir($pluginBasePath . '/Resources/views/');
```

Comment:
as you are in the base plugin class, you can use `$this->getPath()`
no need for getting the container parameter

Reply:
sure. my fault